kvm/v15.0.0/release.yaml

```
@@ -0,0 +1,57 @@
+# Generated with:
+# devctl release create --name v15.0.0 --provider kvm --base 14.1.2 --app cert-exporter@1.7.1 --app chart-operator@2.18.0 --app coredns@1.6.0@1.8.3 --app net-exporter@1.10.2 --component kubernetes@1.20.9 --component calico@3.19.1 --component kvm-operator@update_calico --overwrite
+apiVersion: release.giantswarm.io/v1alpha1
+kind: Release
+metadata:
+  creationTimestamp: null
+  name: v15.0.0
+spec:
+  apps:
+  - name: cert-exporter
+    version: 1.7.1
+  - name: chart-operator
+    version: 2.18.0
```

```suggestion
    version: 2.18.1
```

@yulianedyalkova Could you use this patch release?

Its for this PM https://github.com/giantswarm/giantswarm/issues/18324

It just increases the memory limit for chart-operator.  In the puma workload clusters the prometheus-operator app often hits the limit as its a large chart with a lot of values.